core/controllers/async_download.py

```diff
@@ -11,27 +12,41 @@
 from core.controllers.download import download
 
 
-def async_download(
-    email_address: str,
-    file_format: str,
-    funds: list[str] | None = None,
-    organisations: list[str] | None = None,
-    regions: list[str] | None = None,
-    rp_start: str | None = None,
-    rp_end: str | None = None,
-    outcome_categories: list[str] | None = None,
+def trigger_async_download(
+    email_address,
+    file_format,
+    funds=None,
+    organisations=None,
+    regions=None,
+    rp_start=None,
+    rp_end=None,
+    outcome_categories=None,
 ):
```

Comment:
Is there a reason we've dropped the type hints here?

Reply:
Yes, it seems I haven't spent enough time with mypy and type-hints yet.  ;-)
Added type-hints to the args.